src/routes/admin/domains/all.test.ts

```diff
@@ -4,13 +4,38 @@ import { getAllDomains } from './all'
 describe('Domains - All', () => {
   let resSpy
   let req
+  let reqwithError
+  let reqwithCount
   beforeEach(() => {
     resSpy = createSpyObj('Response', ['status', 'json', 'end', 'send'])
     req = {
       db: { domains: { get: jest.fn() } },
       query: { }
     }
     jest.spyOn(req.db.domains, 'get').mockResolvedValue([])
+    reqwithError = {
```

Comment:
i dont think we need new objects for this -- just overwrite/add whats need for the specific test